herbert_core/utilities/classes/@IX_dataset_3d/sliceomatic.m

```diff
@@ -26,100 +29,173 @@
 % - To set the default for future Sliceomatic sessions -
 %      On the 'Object_Defaults' menu select 'Slice Color Texture'
 
-arglist=struct('name','',...
+
+% Check input arguments
+% ---------------------
+arglist=struct('name','Sliceomatic',...
     'x_axis','x-axis',...
     'y_axis','y-axis',...
     'z_axis','z-axis',...
     'isonormals',0);
 flags={'isonormals'};
+[par,keyword] = parse_arguments(varargin,arglist,flags);
 
-[par,keyword,present] = parse_arguments(varargin,arglist,flags);
-
-% Check input arguments
-% ---------------------
 if ~isempty(par)
     error('HERBERT:IX_dataset_3d:invalid_argument', ...
-        'Invalid sliceomatic arguments:\n %s',disp2str(par))
+        'Invalid sliceomatic arguments given. Only keyword arguments are permitted')
 end
```

Comment:
I would disagree with this simplification. Previous state was showing what is provided invalid, The new one just infoms about some "keywords aruments".

Reply:
I made the change because disp2str(par) can create huge output if e.g. par is a 1000x1000 numeric array it produces thousands of lines of print out, which in the error message would equate to thousands of red lines in the command window. It was that problem i wanted to avoid. However, looking more carefully at the class of variable `par` it is always a row cell array, so in fact it will produce limited amount of output. Accordingly, I'll change the line back to the original in the code.

Reply:
I am moving suggested solution to this problem to ticket #1875